content/pt-br/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Gerenciando Secret usando Arquivo de Configuração
+content_type: task
+weight: 20
+description: Criando objetos Secret usando arquivos de configuração de recursos.
+---
+
+<!-- overview -->
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## Crie o arquivo de configuração
+
+Você pode criar um Secret primeiramente em um arquivo, no formato JSON ou YAML, e depois
```

Comment:
do we have a definition of um secret or uma secret? I'm ok following with "um secret" (as the secret translation is "segredo" which is a masculine word), but just want to check the consistency over the other docs

Reply:
Just to let you know, I found some others use of 'um secret' at the reference docs in pt-br, so I guess it's okay continue with this translation.